src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -372,13 +376,13 @@ class TSPSchoolSyncer extends mix(Syncer).with(ClassImporter) {
 	 * @async
 	 */
 	async updateStudent(user, tspStudent) {
-		const equal = user.firstName === tspStudent.schuelerVorname && user.lastName === tspStudent.schuelerNachname;
-		if (equal) {
-			this.stats.users.students.unchanged += 1;
-			return user;
-		}
-
 		try {
```

Comment:
If the error occurres in this block it means that user or tcpStudent are null or undefined. I would make check for undefined instead.